### ISSUE TITLE
Unable to stop VM in suspended state

### ISSUE THREAD

**Issue by pekof** [CanonicalLtd/multipass#491]
Unable to stop VM in suspended state
```
pekof@ubuntu:~$ multipass list
Name                    State             IPv4             Release
fdr                     SUSPENDED         --               Not Available
pekof@ubuntu:~$ multipass stop fdr -vvv
Stopping fdr /[2018-11-14T10:45:49.320] [info] [fdr] Ignoring shutdown issued while suspended

```

**Comment by Saviq** [comment]
What would you expect to happen in this case? That we discard the saved state? There's a good chance that would corrupt the file system around any open files, also lose any data that you didn't save.

Ultimately we thought a suspended instance should be considered stopped, so the command is a NOOP.

**Comment by pekof** [comment]
I wanted to change driver and I could not until all VMs were stopped.
Discard saved state seems like ok solution in this case.

**Comment by Saviq** [comment]
We would need a fail-safe on it to prevent accidental data loss. Also as mentioned, discarding state could very well result in filesystem issues.

I suppose we could start the instance and shut it down again as soon as it restores. We could add a `--discard-state` option, too — doesn't seem like the `stop` command would be the right place for it, though.

**Comment by Saviq** [comment]
Duplicate of #988
